python/dataproc_templates/pubsublite/README.md

```diff
@@ -0,0 +1,70 @@
+* `pubsublite.to.bq.output.dataset`: BigQuery output dataset name
+* `pubsublite.to.bq.output.table`: BigQuery output table name
+* `pubsublite.to.bq.temp.bucket.name`: Temporary bucket for the Spark BigQuery connector
+
+## Usage
+
+```
+$ python main.py --template PUBSUBLITETOBQ --help
+usage: main.py --template PUBSUBLITETOBQ [-h] \
+    --pubsublite.to.bq.input.subscription PUBSUBLITE.BIGQUERY.INPUT.SUBSCRIPTION \
+    --pubsublite.to.bq.project.id PUBSUBLITE.BIGQUERY.OUTPUT.PROJECT.ID \
+    --pubsublite.to.bq.output.dataset PUBSUBLITE.BIGQUERY.OUTPUT.DATASET \
+    --pubsublite.to.bq.output.table PUBSUBLITE.BIGQUERY.OUTPUT.TABLE \
+    --pubsublite.to.bq.temp.bucket.name PUBSUBLITE.BIGQUERY.TEMP.BUCKET.NAME \
+    [--pubsublite.to.bq.write.mode {overwrite,append,ignore,errorifexists}] \
+    --pubsublite.to.bq.checkpoint.location PUBSUBLITE.BIGQUERY.CHECKPOINT.LOCATION
+```
+
+## Required JAR files
+
+* [Pubsublite Spark SQL Streaming](https://central.sonatype.com/artifact/com.google.cloud/pubsublite-spark-sql-streaming/1.0.0) 
+* [Spark BigQuery connector](https://cloud.google.com/dataproc-serverless/docs/guides/bigquery-connector-spark-example)
+
+## Example submission
+
+```
+export GCP_PROJECT=<project_id>
+export REGION=<region>
+export GCS_STAGING_LOCATION=<gcs-staging-bucket-name> 
+export JARS="gs://spark-lib/pubsublite/pubsublite-spark-sql-streaming-LATEST-with-dependencies.jar,gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar"
+./bin/start.sh \
+-- --template=PUBSUBLITETOBQ \
+--pubsublite.to.bq.input.subscription=<pubsublite input subscription (Format: projects/{input_project_id}/locations/{region}/subscriptions/{input_subscription_name})> \
+--pubsublite.to.bq.project.id=<BQ project ID> \
+--pubsublite.to.bq.output.dataset=<BQ dataset name> \
+--pubsublite.to.bq.output.table=<BQ table name> \
+--pubsublite.to.bq.write.mode=<Output write mode> \
+--pubsublite.to.bq.temp.bucket.name=<temp bucket name> \
+--pubsublite.to.bq.checkpoint.location=<checkpoint folder location>
+```
+
+
+## Configurable Parameters
+
+Following properties are available in commandline or [template.constants](../util/template_constants.py) file:
+
+```
+## Project that contains the input Pub/Sub subscription to be read
+pubsubliteinput.project.id=<pubsublite project id>
+## PubSublite subscription name
+pubsubliteinput.subscription=<pubsublite subscription (Format: projects/{input_project_id}/locations/{region}/subscriptions/{input_subscription_name})>
+## Stream timeout, for how long the subscription will be read
+pubsublitetimeout.ms=120000
+## Streaming duration, how often wil writes to BQ be triggered
+pubsublitestreaming.duration.seconds=15
+## Project that contains the output table
+pubsublitebq.output.project.id=<pubsublite to bq output project id>
+## BigQuery output dataset
+pubsublitebq.output.dataset=<bq output dataset>
+## BigQuery output table
+pubsublitebq.output.table=<bq output table>
```

Comment:
Please correct the template property names

Reply:
Done